packages/shared/src/components/profile/Header.tsx

```diff
@@ -24,7 +25,11 @@ export function Header({
 }: HeaderProps): ReactElement {
   const [, onShareOrCopyLink] = useShareOrCopyLink({
     text: `Check out ${user.name}'s profile on daily.dev`,
-    link: user.permalink,
+    link: addTrackingQueryParams({
+      link: user.permalink,
+      userId: user.id,
+      cid: 'share_profile',
```

Comment:
This one I guess we could also abstract.
Maybe something like we do for AuthEvents?